crypto/nrf_oberon/include/ocrypto_curve_p256.h

```diff
@@ -15,38 +23,75 @@ extern "C" {
 
 
 // (x,y) only jacobian coordinates
+/**@cond */
 typedef struct {
     ocrypto_mod_p256 x;
     ocrypto_mod_p256 y;
 } ocrypto_cp_p256;
+/**@endcond */
 
-
-// load r.x from bytes, recover r.y
-// returns 0 if r is a legal curve point, -1 otherwise
+/** load r.x from bytes, recover r.y
+ *
+ * @param[out]      r       Point with r.x loaded, r.y kept
```

Comment:
This seems to contradict the brief. Is r.y recovered or kept by this function?

Reply:
Made this clearer